packages/components/src/internal/components/samples/constants.ts

```diff
@@ -205,13 +205,13 @@ export const SAMPLE_INSERT_EXTRA_COLUMNS = [...AMOUNT_AND_UNITS_COLUMNS, ...SAMP
 export const SAMPLE_EXPORT_CONFIG = {
     'exportAlias.name': DEFAULT_SAMPLE_FIELD_CONFIG.label,
     'exportAlias.aliquotedFromLSID': ALIQUOTED_FROM_COL,
+    'exportAlias.aliquotedFromLSID/name': ALIQUOTED_FROM_COL,
     'exportAlias.sampleState': STATUS_COL,
     'exportAlias.storedAmount': 'Amount',
 };
 
 export const SAMPLE_DATA_EXPORT_CONFIG = {
```

Comment:
seems like this constant can go away now since it's no different than `SAMPLE_EXPORT_CONFIG`.